src/lib/index-utils.js

```
@@ -0,0 +1,848 @@
+
+/**
+ * Quantize a burndown chart date to 1-hour resolution
+ * @param {Date} date
+ * @param {string} resolution One of 'days', 'hours', 'minutes', 'seconds'
+ * @return {Date} The quantized dates
+ */
+function normaliseDate(date, resolution = 'minutes') {
+  const result = new Date(date.getTime());
+  switch (resolution) {
+    case 'days':
+      result.setHours(0);
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)